app/views/components/SubHeading.scala.html

```diff
@@ -14,11 +14,13 @@
  * limitations under the License.
  *@
 
-@(
-messageKey: String = "",
-inputType: String = "button"
-)(implicit messages: Messages)
+@this()
 
-<div class="section">
-    <input type="@inputType" class="button" value="@messages(messageKey)" />
-</div>
+@(key: String, param: String, hiddenText: Boolean = true)(implicit messages: Messages)
+
+<span class="govuk-caption-l">
+ @if(hiddenText){
+  <span class="govuk-visually-hidden">@messages(s"$key.hidden")</span>
```

Comment:
The message key for this is not defined in messages.en or .cy

Reply:
Component not used. Updated to match SubHeading component within register-trust-other-individual-frontend:
https://github.com/hmrc/register-trust-other-individual-frontend/blob/master/app/views/components/SubHeading.scala.html